### PR TITLE
View house

### PR DESCRIPTION
**Pull request by frmad**

### CODE REVIEW

**Review by Nolle10 — APPROVED** [review]
LGTM

**Review by mathiasgredal — CHANGES_REQUESTED** [review]
Can you add a new year button to the main scene

**Comment by Nolle10** [comment]
> Can you add a new year button to the main scene

There is no need, since we have one inside of the house

File: src/main/java/worldofzuul/presentation/ViewHouseController.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        GUI_Main.setRoot("house");
+    }
+
+    @FXML
+    private void handleNextYear() throws Exception{
+        GUI_Main.setRoot("next year");
+    }
+
+
+
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/ViewHouseController.java:47]: Why these blank lines?

File: src/main/java/worldofzuul/presentation/ViewHouseController.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            imageviewCar2.setOpacity(1.0);
+            electric.setOpacity(1.0);
+            petrol.setOpacity(0.01);
+        }
+
+        if (Game.instance.getHouse().getHeater() instanceof HeatPump) {
+            imageviewPump2.setOpacity(1.0);
+            heatingPump.setOpacity(1.0);
+        }
+
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/ViewHouseController.java:33]: Why the blank line?

**Review by mathiasgredal — APPROVED** [review]
LGTM

**Pull request merged**